bitnami/thanos/templates/alert-rule/replicate.yml

```diff
@@ -34,9 +34,9 @@ spec:
         summary: Thanos Replicate is failing to run.
       expr: |
         (
-          sum by (job) (rate(thanos_replicate_replication_runs_total{result="error", job=~".*thanos-bucket-replicate.*"}[5m]))
+          sum by (job) (rate(thanos_replicate_replication_runs_total{result="error", job=~".*{{ template "common.names.fullname" . }}-replicate.*"}[5m]))
```

Comment:
Could you please review if it is right instead of `job=~".*{{ template "common.names.fullname" . }}-bucket-replicate.*"}`? Maybe adding a new function to `_helpers.tpl` could be a good idea.

Reply:
Based on the previous value, you're right, it should be `job=~".*{{ template "common.names.fullname" . }}-bucket-replicate.*"}` ! I 'll fixed it